From 6969f2f58f4f3b4defd925b0a721b0dd6bb0f611 Mon Sep 17 00:00:00 2001
From: Yu Yue <yuyue@mprc.pku.edu.cn>
Date: Tue, 1 Nov 2011 13:48:24 +0800
Subject: [PATCH 057/641] UniCore64: open config GENERIC_IRQ_SHOW

Fixed error:
fs/built-in.o:(.rodata+0x5440): undefined reference to `show_interrupts'

Signed-off-by: Yu Yue <yuyue@mprc.pku.edu.cn>
---
 arch/unicore64/Kconfig |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/unicore64/Kconfig b/arch/unicore64/Kconfig
index 3ab4632..5e870dd 100644
--- a/arch/unicore64/Kconfig
+++ b/arch/unicore64/Kconfig
@@ -6,6 +6,7 @@ config UNICORE64
 	select HAVE_KERNEL_LZO
 	select HAVE_KERNEL_LZMA
 	select ARCH_WANT_FRAME_POINTERS
+	select GENERIC_IRQ_SHOW
 	help
 	  UniCore-64 is 64-bit Instruction Set Architecture,
 	  including a series of low-power-consumption RISC chip
-- 
1.7.9.5

